api/v1beta1/hetznerbaremetalhost_types.go

```diff
@@ -161,6 +161,9 @@ const (
 	// StateRegistering means we are getting hardware details.
 	StateRegistering ProvisioningState = "registering"
 
+	// StatePreProvisioning means we install a new image.
```

Comment:
comment is the same as for image installing

Reply:
fixed